doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -0,0 +1,604 @@
+-------------------------------
+
+.. note::
+   Skip this step if you are using a Kaby Lake (KBL) NUC.
+
+Go to **Chipset** -> **System Agent (SA) Configuration** -> **Graphics
+Configuration** and make the following settings:
+
+The **DVMT Pre-Allocated** to **64MB**:
+
+.. figure:: images/DVMT-reallocated-64mb.png
+
+Set **PM Support** to **Enabled**: 
+
+.. figure:: images/PM-support-enabled.png
+
+OVMF for User VM launching
+--------------------------
+
+The User VM will be launched by OVMF, so copy it to the specific folder:
+
+.. code-block:: none
+
+   $ sudo mkdir -p /usr/share/acrn/bios
+   $ sudo cp /home/acrn/work/acrn-hypervisor/devicemodel/bios/OVMF.fd  /usr/share/acrn/bios
+
+Install IASL in Ubuntu for User VM launch
+-----------------------------------------
+
+ACRN uses ``iasl`` to parse **User VM ACPI** information. The original ``iasl``
+in Ubuntu 18.04 is too old to match with ``acrn-dm``; update it using the
+following steps:
+
+.. code-block:: none
+
+   $ sudo -E apt-get install iasl bison flex
+   $ cd /home/acrn/work
+   $ wget https://acpica.org/sites/acpica/files/acpica-unix-20191018.tar.gz
+   $ tar zxvf acpica-unix-20191018.tar.gz
+   $ cd acpica-unix-20191018
+   $ make clean && make iasl
+   $ sudo cp ./generate/unix/bin/iasl /usr/sbin/
+
+Launch the RTVM
+***************
+
+Update the Grub file
+====================
+
+#. Update the ``/etc/grub.d/40_custom`` file as shown below.
+
+   .. note::
+      Enter the command line for the kernel in ``/etc/grub.d/40_custom`` as
+      a single line and not as multiple lines. Otherwise, the kernel will
+      fail to boot.
+
+   **menuentry 'ACRN Ubuntu User VM' --id ubuntu-user-vm**
+
+   .. code-block:: bash
+
+      {
+
+        load_video
+        insmod gzio
+        insmod part_gpt
+        insmod ext2
+        set root=hd0,gpt2
+
+        search --no-floppy --fs-uuid --set b2ae4879-c0b6-4144-9d28-d916b578f2eb
+        echo 'loading ACRN...'
+
+        linux  /boot/bzImage root=root=PARTUUID=<UUID of rootfs partition> rw rootwait nohpet console=hvc0 console=ttyS0 no_timer_check ignore_loglevel log_buf_len=16M consoleblank=0 clocksource=tsc tsc=reliable x2apic_phys processor.max_cstate=0 intel_idle.max_cstate=0 intel_pstate=disable mce=ignore_ce audit=0 isolcpus=nohz,domain,1 nohz_full=1 rcu_nocbs=1 nosoftlockup idle=poll irqaffinity=0
+
+      }
+
+   .. note::
+
+      Update this to use your uuid and PARTUUID for the root= parameter (or
+      use the device node directly).
+
+      Update the kernel name if you used a different name as the source
+      for your Service VM kernel.
+
+#. Modify the ``/etc/default/grub`` file to make the grub menu visible when
+   booting and make it load the Service VM kernel by default. Modify the
+   lines shown below:
+
+   .. code-block:: none
+
+      GRUB_DEFAULT=ubuntu-user-vm
+      #GRUB_TIMEOUT_STYLE=hidden
+      GRUB_TIMEOUT=5
+
+#. Update Grub on your system:
+
+   .. code-block:: none
+
+      sudo update-grub
```

Comment:
```suggestion
      $ sudo update-grub
```

Reply:
Yes.
@deb-intel Please helps on this.
Thanks!

Reply:
@fuzhongl Are you referring to the question that @gvancuts asked (Can we add a note giving the user a clue that this is not the same?)

If so, I could use your help. What do I need to say?

Reply:
@deb-intel  I don't think it is necessary to add note. Because we add **Build and Installation RT kernel for Ubuntu User VM** at above step.
 @gvancuts what's your opinion?

Reply:
> @deb-intel I don't think it is necessary to add note. Because we add **Build and Installation RT kernel for Ubuntu User VM** at above step.
> @gvancuts what's your opinion?

I agree, with the new instructions above this is no longer needed. Just make sure that the instructions match this line, i.e. the `bzImage` gets put in the right place.